### PR TITLE
Feat: sign message dialog

### PR DESCRIPTION
**Pull request by jhasuraj01**
- feat: message and json dialog ready
- fix: removed unwanted file
- feat: added changeset

### CODE REVIEW

**Comment by jhasuraj01** [comment]
Screenshot_20230901_173027
Screenshot_20230901_173040
Screenshot_20230901_173103

File: packages/cysync-core/src/dialogs/SignMessage/Dialogs/ViewMessage.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+              <Flex gap={8} align="center">
+                <CoinIcon assetId={account.assetId} size={16} />
+                <Typography variant="span" $fontWeight="medium">
+                  <LangDisplay text={account.name} />
+                </Typography>
+              </Flex>
+            </Flex>
+          </Container>
+          <ScrollContainer
+            style={{ background: 'none', paddingLeft: 0, paddingRight: 0 }}
```
Comment (irshadCypherock) [inline packages/cysync-core/src/dialogs/SignMessage/Dialogs/ViewMessage.tsx]: Don't use style, use props existing props

File: packages/cysync-core/src/dialogs/SignMessage/Dialogs/ViewJson.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+            >
+              <Typography color="muted">
+                <JsonView
+                  src={JSON.parse(json ?? '{}')}
+                  enableClipboard={false}
+                  displayObjectSize={false}
+                  displayDataTypes={false}
+                  indentWidth={4}
+                  collapsed={false}
+                  style={{ background: 'none' }}
```
Comment (irshadCypherock) [inline packages/cysync-core/src/dialogs/SignMessage/Dialogs/ViewJson.tsx]: Change this in `ui` package

File: packages/cysync-core/src/dialogs/SignMessage/Dialogs/ViewJson.tsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+              <Flex gap={8} align="center">
+                <CoinIcon assetId={account.assetId} size={16} />
+                <Typography variant="span" $fontWeight="medium">
+                  <LangDisplay text={account.name} />
+                </Typography>
+              </Flex>
+            </Flex>
+          </Container>
+          <ScrollContainer
+            style={{ background: 'none', paddingLeft: 0, paddingRight: 0 }}
```
Comment (irshadCypherock) [inline packages/cysync-core/src/dialogs/SignMessage/Dialogs/ViewJson.tsx]: Use props

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: packages/cysync-core/src/dialogs/SignMessage/Dialogs/ViewJson.tsx (hunk 3 above)
Comment (jhasuraj01) [reply packages/cysync-core/src/dialogs/SignMessage/Dialogs/ViewJson.tsx]: Added utils props!

File: packages/cysync-core/src/dialogs/SignMessage/Dialogs/ViewMessage.tsx (hunk 1 above)
Comment (jhasuraj01) [reply packages/cysync-core/src/dialogs/SignMessage/Dialogs/ViewMessage.tsx]: Added utils props!

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**